utils/deploys/deployUtils.ts

```diff
@@ -525,4 +531,60 @@ export async function updateSetManager(
     });
     await writeTransactionToOutputs(setManagerTransaction.transactionHash, description);
   }
+}
+
+async function getEnvironment(hre: HardhatRuntimeEnvironment) {
+  const [owner] = await getAccounts();
+  const instanceGetter = new InstanceGetter(owner.wallet);
+  const { rawTx } = hre.deployments;
+  const { deployer } = await hre.getNamedAccounts();
```

Comment:
Where we grab the `deployer`, comparable to [set-v2-strategies](https://github.com/SetProtocol/set-v2-deployments/blob/599859a9dff846dc933253f7fe3f27e6e7d41c27/utils/deployUtils.ts#L29)